pineappl/src/grid.rs

```diff
@@ -1384,6 +1384,8 @@ impl Grid {
         }
     }
 
+    /// TODO: Fix to account for the different EKOs
+    /// TODO: Does the info have to be different?
```

Comment:
yes, they will be different - there has to be one info for one EKO; e.g. there are typically less polarized flavor than unpolarized

Reply:
So, practically, what would be different in each case from the following attributes?
https://github.com/NNPDF/pineappl/blob/2a8a9912a29d81d8b4ca4906eeec8daf6f5f80ea/pineappl_py/src/grid.rs#L474-L486